examples/acquiring_data/downloading_cutouts.py

```diff
@@ -0,0 +1,93 @@
+"""
+===============================================
+Requesting cutouts of AIA images from the JSOC
+===============================================
+
+This example shows how to request a cutout of a series of
+AIA images from the JSOC and animate the resulting sequence.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.time
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.visualization import ImageNormalize, SqrtStretch
+
+import sunpy.map
+from sunpy.net import Fido, attrs, jsoc
```

Comment:
We should stick to the import conventions, also I don't think we should use the `JSOCClient` directly?

```suggestion
from sunpy.net import Fido, attrs as a
```